CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 
-- Added this changelog file
+- (@hisaac) Added this changelog file
+- (@ty-wilson) Fixed issue #72 changing the default value on Apple Events to "Allow"
+- (@watkyn) Added support for the new Authorization key in Big Sur
```

Comment:
Should add the fact that this will run only on macOS 10.15+ now.

Reply:
done

Reply:
We should also change it in the readme. It currently says it works on 10.13 or newer.